xonsh/history.py

```diff
@@ -287,16 +288,16 @@ def _create_parser():
     if _HIST_PARSER is not None:
         return _HIST_PARSER
     from argparse import ArgumentParser
-    p = ArgumentParser(prog='history', 
+    p = ArgumentParser(prog='history',
                        description='Tools for dealing with history')
     subp = p.add_subparsers(title='action', dest='action')
     # show action
     show = subp.add_parser('show', help='displays current history, default action')
     show.add_argument('-r', dest='reverse', default=False, action='store_true',
                       help='reverses the direction')
-    show.add_argument('n', nargs='?', default=None, 
-                      help='displays n current history entries, n may be an int or use '
-                           'Python slice notation')
+    show.add_argument('n', nargs='?', default=None,
+                      help='display n\'th history entry if n is a simple int, '
```

Comment:
Nope, it's supposed to be "n'th" as in "1st", "2nd", "3rd", "4th", etc. The original help implies that "history show 5" shows the five most recent history entries when in fact it shows #5 (i.e., the sixth).